.docs/README.md

```diff
@@ -54,13 +54,14 @@ extensions:
     orm.annotations: Nettrine\ORM\DI\OrmAnnotationsExtension
 
 orm.annotations:
-    paths:
+    mapping:
         # your app entities
-        - App/Model/Database/Entity
+        App\Model\Database\Entity: '%appDir%/Model/Database/Entity'
         # doctrine extensions entities
-        - Gedmo\Loggable\Entity
-        - Gedmo\Loggable\Entity
-        - Gedmo\Tree\Entity
+        Gedmo\Translatable: '%appDir%/../vendor/gedmo/doctrine-extensions/lib/Gedmo/Translatable/Entity'
```

Comment:
Jeste prosim odebrat ty ' kolem hodnot.

Reply:
ty tam nemusí být? :o

Reply:
Zkousel jsi to bez nich? Myslim, ze nemusi. :-)

Reply:
upraveno